src/main/java/seedu/address/storage/JsonAdaptedAppointment.java

```diff
@@ -17,16 +17,18 @@ class JsonAdaptedAppointment {
 
     private final String description;
     private final LocalDateTime dateTime;
+    private final String name;
```

Comment:
Change field to doctorName to make it clearer

Reply:
yes agreed